src/main/java/com/notably/logic/commands/DeleteCommand.java

```diff
@@ -16,6 +19,8 @@
 public class DeleteCommand extends Command {
     public static final String COMMAND_WORD = "delete";
     public static final String COMMAND_SHORTHAND = "d";
+
+    private final Logger logger = LogsCenter.getLogger(getClass());
```

Comment:
Oops sorry missed this before. Just a tiny thing. Let's make this `logger` a `private static final` field since we don't need to create one for each instance of `DeleteCommand`!